browser/components/MarkdownPreview.js

```diff
@@ -127,13 +127,21 @@ h1, h2 {
   border: none;
 }
 
+h3 {
+  margin: 1em 0 0.4em;
```

Comment:
Can you change this to `1em 0 0.8em`?
```suggestion
  margin: 1em 0 0.8em;
```